From 8548ca5c9ea6d38ec15840b38d69a32bf10a9cc7 Mon Sep 17 00:00:00 2001
From: ragusaa <54862477+ragusaa@users.noreply.github.com>
Date: Thu, 19 Oct 2023 17:45:43 -0400
Subject: [PATCH] ClamOnAcc: Fix infinite loop when OnAccessIncludePath doesn't
 exist

---
 clamonacc/inotif/inotif.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/clamonacc/inotif/inotif.c b/clamonacc/inotif/inotif.c
index 8fc1b3dd20..2aeb98136f 100644
--- a/clamonacc/inotif/inotif.c
+++ b/clamonacc/inotif/inotif.c
@@ -598,8 +598,8 @@ void *onas_ddd_th(void *arg)
                         }
                     }
                 }
-                pt = (struct optstruct *)pt->nextarg;
             }
+            pt = (struct optstruct *)pt->nextarg;
         }
     }
 

